frontend/src/components/Alert.tsx

```diff
@@ -0,0 +1,16 @@
+import { Alert as USWDSAlert } from "@trussworks/react-uswds";
+
+type Props = {
+  type: "success" | "warning" | "error" | "info";
+  children: React.ReactNode;
+};
+
+const Alert = ({ type, children }: Props) => {
+  return (
+    <USWDSAlert type={type} headingLevel="h4" slim>
```

Comment:
My first time working USWDS design library, and it's turning out to be awesome. Love we can just drop this in.